### PR TITLE
Us 11

### PR DESCRIPTION
**Pull request by Ahmtfrks**

### CODE REVIEW

File: src/test/java/com/fleetmanagement/stepDefinitions/LoginStepDefinitions.java [hunk 1]
```diff
@@ -36,16 +36,15 @@ public void theUserShouldSeeTheDashboard() {
     @When("the user enters Store Manager information")
     public void theUserEntersStoreManagerInformation() {
         loginPage.login("storemanager51", "UserUser123");
-        BrowserUtils.waitForInvisibilityOf(loginPage.loadingBar);
+
```
Comment (mujganaydin) [inline src/test/java/com/fleetmanagement/stepDefinitions/LoginStepDefinitions.java:39]: Why did you delete waitForInvisibilityOf() method here?

Comment (Ahmtfrks) [reply src/test/java/com/fleetmanagement/stepDefinitions/LoginStepDefinitions.java:39]: beacuse its not needed we use that before
